aws/resource_aws_autoscaling_notification_test.go

```diff
@@ -211,19 +211,28 @@ func testAccCheckAWSASGNotificationAttributes(n string, asgn *autoscaling.Descri
 }
 
 func testAccASGNotificationConfig_basic(rName string) string {
-	return fmt.Sprintf(`
+	return testAccLatestAmazonLinuxHvmEbsAmiConfig() + fmt.Sprintf(`
 resource "aws_sns_topic" "topic_example" {
   name = "user-updates-topic-%s"
 }
 
 resource "aws_launch_configuration" "foobar" {
   name          = "foobarautoscaling-terraform-test-%s"
-  image_id      = "ami-21f78e11"
-  instance_type = "t1.micro"
+  image_id      = data.aws_ami.amzn-ami-minimal-hvm-ebs.id
+  instance_type = "t2.micro"
+}
+
+data "aws_availability_zones" "available" {
+  state = "available"
+
+  filter {
+    name   = "opt-in-status"
+    values = ["opt-in-not-required"]
+  }
 }
 
 resource "aws_autoscaling_group" "bar" {
-  availability_zones        = ["us-west-2a"]
+  availability_zones        = [data.aws_availability_zones.available.names[1]]
```

Comment:
Out of curiosity, any reason to pick the second AZ name instead of the first? Shouldn't matter since most AWS regions except Tokyo are 2+ AZs, but might be good to stick with the first AZ should we need to test in those special regions.

Reply:
Yes, there is no `t1.micro` in `us-gov-west-1a` and many tests use that instance type. `us-gov-west-1b` and `us-west-2b` both have `t1.micro`.

Reply:
Gotcha, we'll get these over to the `aws_ec2_instance_type_offering` data source soon enough. 👍